packages/core/src/components/utils/modal/message.ts

```diff
@@ -53,25 +53,23 @@ function createConfirmButtons(
       text: textOkay,
       type: 'okay',
       payload: payloadOkay,
-    },
+    } as MessageAction,
   ];
 }
 
-export type MessageConfig<T> =
-  | Omit<ModalConfig<T, unknown>, 'content' | 'title'>
-  | MessageContent;
```

Comment:
Why did you removed this type? Its a good idea in general to have a MessageConfig which reflects the ModalConfig